generators/uimodule/platform.js

```diff
@@ -0,0 +1,147 @@
+import dependencies from "../dependencies.js"
+import { ensureCorrectDestinationPath } from "../helpers.js"
+import fs from "fs"
+import Generator from "yeoman-generator"
+import yaml from "yaml"
+
+export default class extends Generator {
+	writing() {
+		ensureCorrectDestinationPath.call(this)
+
+		const uimodulePackageJson = JSON.parse(fs.readFileSync(this.destinationPath("package.json")))
+		const manifestJSON = JSON.parse(fs.readFileSync(this.destinationPath("webapp/manifest.json")))
+
+		const platformIsWebserver = this.options.config.platform === "Static webserver"
+		const platformIsApprouter = this.options.config.platform === "Application Router"
+		const platformIsHTML5AppsRepo = this.options.config.platform === "SAP HTML5 Application Repository Service"
+		const platformIsSAPBuildWorkZone = this.options.config.platform === "SAP Build Work Zone, standard edition"
+		const platformIsSAPNetWeaver = this.options.config.platform === "SAP NetWeaver"
+
+		delete uimodulePackageJson.scripts["deploy"]
+		delete uimodulePackageJson.scripts["deploy-config"]
+
+		// we do have a .gitignore at root level
+		fs.unlinkSync(this.destinationPath(".gitignore"))
+
+		if (platformIsWebserver) {
```

Comment:
`switch` might add to readability here...

Reply:
Agree